sql/planbuilder/scope.go

```diff
@@ -152,7 +152,7 @@ func (s *scope) triggerCol(table, col string) (scopeColumn, bool) {
 	// hallucinate tablecol
 	for _, t := range s.b.TriggerCtx().UnresolvedTables {
 		if strings.EqualFold(t, table) {
-			col := scopeColumn{table: t, col: col}
+			col := scopeColumn{tableId: sql.NewAliasID(table), col: col}
```

Comment:
not sure it matters, but I think we're assuming that this hallucinated table would have to be in the default database at call time. could be weird edge cases with a column reference using `db.table.col`, which errors because of this but might be valid at runtime